applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -69,6 +73,9 @@ enum state {
 
 #define AXIS_COUNT (IS_ENABLED(CONFIG_DESKTOP_HID_REPORT_MOUSE_SUPPORT) * MOUSE_REPORT_AXIS_COUNT)
 
+static const uint8_t handled_output_reports[] = {
```

Comment:
I preferred to make it internal for HID state - we define only HID output reports that are handled by the HID state here (in generic case it could be only subset of HID output reports). 
If you prefer to have it in `hid_report_desc.h`, I can move it.

Reply:
I prefer to move it for the sake of consistency. It looks to me as a sibling of input_reports array (when moving rename it to output_reposts) that is present there.
Expect it will be optimized out when unused in both cases.